notebooks/3.jump-analysis/nbconverted/3.jump_analysis.py

```diff
@@ -423,28 +426,78 @@
 # In[19]:
 
 
+# setting column arrangement
+col_arrangement = [
+    "Metadata_Plate",
+    "Plate_Map_Name",
+    "Metadata_Well",
+    "Metadata_pert_type",
+    "Metadata_gene",
+    "Metadata_pert_iname",
+    "Metadata_target_sequence",
+    "pred_injury",
+    "probability",
+]
+
+# creaeting a rename dict
+rename_assay_type = {
+    "JUMP-Target-1_orf_platemap": "orf",
+    "JUMP-Target-1_compound_platemap": "compound",
+    "JUMP-Target-1_crispr_platemap": "crispr",
+}
+
 # split meta and feature columns
 shared_jump_meta, shared_jump_feats = split_meta_and_features(shared_jump_df)
 
 # selecting the columns
-jump_meta = shared_jump_df[["Metadata_Plate", "Metadata_Well", "Metadata_pert_iname"]]
+predicted_df = shared_jump_df[
+    [
+        "Metadata_Plate",
+        "Metadata_Well",
+        "Metadata_gene",
+        "Metadata_pert_iname",
+        "Metadata_target_sequence",
+        "Metadata_pert_type",
+    ]
+]
 
 # converting injury codes to injury names
-jump_meta["pred_injury"] = [
+predicted_df["pred_injury"] = [
     injury_decoder[str(injury_code)] for injury_code in y_pred.tolist()
 ]
 
 # obtaining the probability score of the predicted injury
-jump_meta["probability"] = y_proba.max(axis=1).tolist()
+predicted_df["probability"] = y_proba.max(axis=1).tolist()
 
+# Merge barcode information by using the Plate ID to indicate the type of treatments applied
+predicted_df = pd.merge(
```

Comment:
I'm not confident that this merge will work correctly...

Don't you want to merge on plate _AND_ well?

Reply:
This was a previous comment that I forgot to edit. The purpose of the merge was to include the assay type for each plate. The barcode_df  contained only the plate ID and the type of assay. Here’s a small example for that `barcode_df` variable. 

| Assay_Plate_Barcode | Plate_Map_Name                  |
|---------------------|---------------------------------|
| BR00117006          | JUMP-Target-1_orf_platemap      |
| BR00118049          | JUMP-Target-1_orf_platemap      |
| BR00118050          | JUMP-Target-1_orf_platemap      |
| BR00118039          | JUMP-Target-1_orf_platemap      |
| BR00118040          | JUMP-Target-1_orf_platemap      |
| BR00117020          | JUMP-Target-1_orf_platemap      |
| BR00117021          | JUMP-Target-1_orf_platemap      |
| BR00117022          | JUMP-Target-1_orf_platemap      |
| BR00117023          | JUMP-Target-1_orf_platemap      |
| BR00118041          | JUMP-Target-1_crispr_platemap   |